string name
geometry_msgs/Pose pose
geometry_msgs/Twist velocity
geometry_msgs/Wrench force
